lib/openstack_query/runners/runner_wrapper.py

```diff
@@ -246,16 +248,23 @@ def _run_query(
         openstacksdk query is run - these kwargs are specific to the resource runner.
         """
 
-    @abstractmethod
     def _parse_subset(
-        self, conn: OpenstackConnection, subset: List[OpenstackResourceObj]
+        self, _, subset: List[OpenstackResourceObj]
     ) -> List[OpenstackResourceObj]:
         """
         This method is a helper function that will check a subset of openstack objects and check their validity
-        :param conn: An OpenstackConnection object - used to connect to openstacksdk
+        :param _: An OpenstackConnection object - not used right now
```

Comment:
If this isn't being used by the method, should we remove this parameter?

Reply:
I guess we can remove it - I was thinking that we'd want to run some sort of validation on each resrouce - something like `find_<resource>` before using it - but I guess it rather slow